pkg/sql/plan/function/func_builtin.go

```diff
@@ -1123,6 +1123,206 @@ func builtInSerial(parameters []*vector.Vector, result vector.FunctionResultWrap
 	return nil
 }
 
+func BuiltInSerialFull(parameters []*vector.Vector, result vector.FunctionResultWrapper, proc *process.Process, length int) error {
+
+	rs := vector.MustFunctionResult[types.Varlena](result)
+	ps := types.NewPackerArray(length, proc.Mp())
+	defer func() {
+		for _, p := range ps {
+			p.FreeMem()
+		}
+	}()
+
+	for _, v := range parameters {
+		if v.IsConstNull() {
+			nulls.AddRange(rs.GetResultVector().GetNulls(), 0, uint64(length))
```

Comment:
if the value is null, you will put the result as `null`？ 
I'm not sure if it's the right meaning for serial_full

Reply:
Thank you. I have fixed it as below.

```go
 if v.IsConstNull() {
	for i := 0; i < v.Length(); i++ {
		ps[i].EncodeNull()
	}
	continue
}
```